packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -530,6 +535,39 @@ const isElementVisible = async (
 	return true;
 };
 
+const clickIfClickable = async (
+	driver,
+	elementLocator,
+	maxIteration = 25,
+	iteration = 0
+) => {
+	const element = await waitForVisible(
+		driver,
+		elementLocator,
+		maxIteration,
+		iteration
+	);
+
+	try {
+		await element.click();
```

Comment:
It would be interesting to return the result of this call, this way we would let the caller to use the element for other purposes.

Reply:
This is updated in this commit: https://github.com/WordPress/gutenberg/pull/41126/commits/86a5d30805990eb22387907c2cf352b3dee8ea21